ckanext/validation/plugin/__init__.py

```diff
@@ -302,6 +307,18 @@ def after_update(self, context, data_dict):
                 del self.resources_to_validate[resource_id]
 
                 _run_async_validation(resource_id)
+    
+    if ckan_2_10:
```

Comment:
This conditional isn't really needed. It's fine to create the functions even if they won't be called.

Reply:
On point. I removed that condition.